src/lib.rs

```diff
@@ -311,23 +318,24 @@ fn run_bindings_exec_subcommand(
                 bail!("No script file and no language specified, so I don't know what language shell to start")
             }
             let script_file_buf = PathBuf::from(script_file.unwrap());
-            let ext = script_file_buf.extension().unwrap_or_default();
-            if ext == "kts" {
-                "kotlin"
-            } else if ext == "py" {
-                "python"
-            } else {
-                bail!("Cannot guess language of script file, please specify it explicitly")
+            match script_file_buf.extension().unwrap_or_default().to_str() {
```

Comment:
Oooooh, yes I see, `to_str()`; I spent like half an hour the other day trying to figure out whether I could `match` on an `OsStr` in a nice way, but this makes a lot of sense :-)